wls-admin-service/src/main/java/de/muenchen/oss/wahllokalsystem/adminservice/client/auth/WahllokalBenutzerClientMapper.java

```diff
@@ -0,0 +1,25 @@
+package de.muenchen.oss.wahllokalsystem.adminservice.client.auth;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.muenchen.oss.wahllokalsystem.adminservice.eai.auth.model.WahllokalUserInfoDTO;
+import de.muenchen.oss.wahllokalsystem.adminservice.service.wahllokalbenutzer.TripleOfWahlbezirkIDWahlnummerWahlIDModel;
+import de.muenchen.oss.wahllokalsystem.adminservice.service.wahllokalbenutzer.WahllokalBenutzerModel;
+import java.util.List;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper
+public interface WahllokalBenutzerClientMapper {
+
+    ObjectMapper objectMapper = new ObjectMapper();
+
+    @Mapping(target = "wbidWahlnummer", source = "wbid_wahlnummer")
+    WahllokalUserInfoDTO toWahllokalUserInfoDTO(WahllokalBenutzerModel wahllokalBenutzerModel);
```

Comment:
gibt es einen grund, dass die objekte `UserInfo` und `Benutzer` heißen? wenn es das gleiche einmal als dto und einmal als model ist sollten sie auch gleich heißen.

```suggestion
    WahllokalUserInfoDTO toWahllokalUserInfoDTO(WahllokalUserInfoModel wahllokalUserInfoModel);
```

Reply:
Verbesserung für phase 3 aus meiner sicht, code und benamung stimmt mit altem code überein, daher sollte man es aktuell nicht anpassen.